rasa_nlu/config.py

```diff
@@ -30,6 +30,7 @@
     "port": 5000,
     "server_model_dirs": None,
     "token": None,
+    "cors_origins": [],
```

Comment:
needs to be added to `config_defaults.json` as well (this will also fix the failing test)